charts/opentelemetry-operator/templates/deployment.yaml

```diff
@@ -37,6 +37,9 @@ spec:
     spec:
       automountServiceAccountToken: {{ .Values.automountServiceAccountToken }}
       hostNetwork: {{ .Values.hostNetwork }}
+      {{- if and (semverCompare ">=1.33-0" .Capabilities.KubeVersion.Version) (kindIs "bool" .Values.hostUsers) }}
```

Comment:
@t3mi based on the docs you linked this has been beta since 1.30. Can we reduce this semver check?

Reply:
Changed to 1.30